packages/platform/api/src/auth/auth.controller.ts

```diff
@@ -35,12 +36,29 @@ export class AuthController {
   @ApiOperation({ summary: 'Validate FusionAuth token' })
   @ApiResponse({ status: 200, description: 'Token successfully validated' })
   @ApiResponse({ status: 401, description: 'Invalid token' })
-  @Post('validate')
-  async validateToken(@Body('token') token: string) {
-    const user = await this.authService.validateUser(token);
-    return this.authService.generateToken(user);
+  @Post('validate-fusion-token')
```

Comment:
let's revert this back to `validate` why did we need to make it `validate-fusion-token`?